packages/http-client/src/common-client-options.ts

```diff
@@ -0,0 +1,90 @@
+import { JSONSchemaType } from "@typespec/compiler";
+
+export interface ClientEmitterOptions {
```

Comment:
How does the docs handle these common options? Will these options reflect in js emitter -
https://tspwebsitepr.z22.web.core.windows.net/prs/6707/docs/emitters/clients/http-client-js/reference/emitter/?

Reply:
yeah you need to also spread those options in your own $lib